be/src/exec/hdfs_scanner_orc.cpp

```diff
@@ -696,6 +695,11 @@ void HdfsOrcScanner::do_update_counter(HdfsScanProfile* profile) {
     COUNTER_UPDATE(stripe_active_lazy_coalesce_together_counter, _app_stats.orc_stripe_active_lazy_coalesce_together);
     COUNTER_UPDATE(stripe_active_lazy_coalesce_seperately_counter,
                    _app_stats.orc_stripe_active_lazy_coalesce_seperately);
+
+    if (_orc_reader != nullptr) {
+        // _orc_reader is nullptr for split task
+        root->add_info_string("ORCSearchArgument: ", _orc_reader->get_search_argument_string());
```

Comment:
suggest use root_profile instead of root, root is a common name, so it is puzzling.

Reply:
changed